grafana-plugin/src/containers/Alerts/Alerts.tsx

```diff
@@ -54,17 +57,33 @@ export const Alerts = function () {
 
   const store = useStore();
   const {
-    userStore: { currentUser },
+    /**
+     * TODO: if we don't dereference currentUserPk then for some reason the @computed portion of
+     * UserStore.currentUser doesn't recalculate and will just be stuck on returning undefined..
+     *
+     * If we dereference currentUserPk here, even if we don't use it.. things just seem to work
+     * (what is this mobx wizardry?)
+     *
+     * Seems to be related to this https://stackoverflow.com/questions/77724466/mobx-computed-not-updating
+     */
+    userStore: { currentUser, currentUserPk },
```

Comment:
@grafana/grafana-oncall-frontend for the life of me I can't figure this out 😆

it works, but if I delete the dereferenced `currentUserPk` (which isn't actually used in this scope), it stops working

Reply:
this actually breaks the frontend build for now

https://github.com/grafana/oncall/actions/runs/10393286285/job/28780417264?pr=4802#step:4:15